src/main/java/sirius/kernel/commons/Strings.java

```diff
@@ -575,17 +575,19 @@ public static String cleanup(@Nullable String inputString, @Nonnull Iterable<Una
         return value;
     }
 
-    protected static final String REGEX_DETECT_XML = "</?[a-zA-Z][^>]*>";
+    protected static final String REGEX_DETECT_XML_TAGS = "</?[a-zA-Z][^>]*>";
+    protected static final String REGEX_DETECT_ENTITIES = "&#?[a-zA-Z0-9]*;";
```

Comment:
```suggestion
    protected static final String REGEX_DETECT_ENTITIES = "&#?[a-zA-Z0-9]+;";
```

Reply:
Would just enforce that something exists between `&` and `;`. Just in case someone uses the char combination `&;` for whatever reason and we detect it as XML.